client/src/components/__snapshots__/Leaderboard.test.js.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders without crashing 1`] = `
+<div
+  className="flexcenter-container"
+>
+  <div
+    className="dashboard-header"
+  >
+    <p
+      className="dashboard-header-text-large"
+    >
+      Volunteer Leaderboard
+    </p>
+  </div>
+  <div
+    className="events-list"
+  >
+    <ul />
+  </div>
+</div>
+`;
```

Comment:
Jest creates a snapshot of the test. It saves the output and when run again, the test will fail if the component changes in some way to the expected test. We commit these changes. 

Useful links:
https://jestjs.io/docs/en/snapshot-testing